docs/geedocs/docsrc/answer/releaseNotes/relNotesGEE5_3_7.rst

```diff
@@ -31,6 +28,11 @@ Release notes: Open GEE 5.3.7
       -  CentOS 6.x and 7.x
       -  Ubuntu 16.04 LTS
 
+      NOTE: CentOS 6 was officially retired in November 2020, and is no
+      longer available for download except through CentOS archives. CentOS 6
+      is not being tested for this release and was not tested in 5.3.6.
```

Comment:
Very minor: It occurred to me that "is not being tested for this release" won't make as much sense in the future when the release is done. Maybe switch to "CentOS 6 was not tested for this release or 5.3.6."?